src/utils/createField.ts

```diff
@@ -43,6 +43,31 @@ export function createSubmit(): HTMLElement {
   return fieldContainer;
 }
 
+/**
+ * Helper for adding icon to background
+ *
+ * @param {HTMLElement} container - container for adding background
+ * @param {string} icon - added icon
+ */
+function addBackgroundIcon(container: HTMLElement, icon: string): void {
+  container.style.backgroundImage = `url(data:image/svg+xml;utf8,${encodeURI(
+    icon
+  )})`;
+}
+
+/**
+ * Helper for creating close button
+ * 
+ * @returns {HTMLButtonElement}
+ */
+export function createClose(): HTMLButtonElement {
+  const closeContainer = make('button', classes.close) as HTMLButtonElement;
+
+  addBackgroundIcon(closeContainer, IconCross);
+
+  return closeContainer;
+}
+
```

Comment:
close button is not a field, seems like this file is not the best place to define `createClose` function.

Reply:
I would suggest moving this logic into Form class, and also put attaching click event listener to the same method